src/functions.php

```diff
@@ -83,16 +81,17 @@ function getFtpConnection($url, $user, $password, $directory, $secure)
 /**
  * upload image files via ftp to the fritzbox fonpix directory
  *
- * @param stdClass[] $vcards downloaded vCards
+ * @param object $vcards downloaded vCards
  * @param array $config
  * @param array $phonebook
  * @param callable $callback
  * @return mixed false or [number of uploaded images, number of total found images]
  */
-function uploadImages(array $vcards, array $config, array $phonebook, callable $callback=null)
```

Comment:
Why remove the array type hint?

Reply:
ping

Reply:
build back a few commits ago -> outdated